Extensions/Lighting/lightruntimeobject-pixi-renderer.js

```diff
@@ -309,6 +319,8 @@ gdjs.LightRuntimeObjectPixiRenderer.prototype._updateBuffers = function () {
     this._defaultVertexBuffer[6] = this._object.x - this._radius;
     this._defaultVertexBuffer[7] = this._object.y - this._radius;
 
+    if(!this._light) return;
+    this._light.shader.uniforms.center = this._center;
```

Comment:
We aren't returning quickly because tests depend upon the buffer data, and if we bail out early, buffer data would not be updated. For now, it's returning after updating the buffer to pass tests, but it's not ideal I would say. If nothing is rendering on the screen then we shouldn't compute any data related to it as well, so we would need to modify the tests somehow 🤔.

Reply:
Not sure I understand, why are tests are impacted? Is WebGL not supported by ChromeHeadless?

Reply:
> HeadlessChrome 85.0.4183 (Windows 10.0.0) WARN: 'This device does not support webgl, which is required for Lighting Extension.'

This is what the HeadlessChrome triggers, so I guess WebGL is not supported then. 😬